Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -19,11 +21,26 @@ if ($scriptArguments -match '[\r\n]') {
     throw (Get-VstsLocString -Key InvalidScriptArguments0 -ArgumentList $scriptArguments)
 }
 
+if($targetAzurePs -eq "OtherVersion") {
+    $targetAzurePs = $customTargetAzurePs.Trim()
+}
+
+$pattern = "^[0-9]+\.[0-9]+\.[0-9]+$"
```

Comment:
can we be explicit in help mark for not supporting regex for version like - 2.3.*, 3.*,.